src/encoded/helpers/helper.py

```diff
@@ -103,8 +103,10 @@ def search_result_actions(request, doc_types, es_results, position=None):
 
 class View_Item:
     def __init__(self, request, search_base):
+        # strip out x.limit=<number> and y.limit=<number>; as they are not used
```

Comment:
this comment just says what the code does.  It's not a bad idea to explain why we're doing this, but it's weird to have a comment at the top of init.  Maybe put the sub in a function and then explain in there.  Just don't explain what the next line of code does unless it's really confusing.

Reply:
@caseylitton I took out the comment. Code supposed to explain itself or be re-written to. With that in mind, the comment was unnecessary as one could read the code.

Reply:
Okay, but again it is not a bad idea to put it in a function and explain why we're doing it.  Just don't explain what the code does.